modules/codegen/src/smithy4s/codegen/SmithyToIR.scala

```diff
@@ -742,6 +740,8 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
         // traits from the synthetic namespace, e.g. smithy.synthetic.enum
         // don't have shapes in the model - so we can't generate hints for them.
         .filterNot(_.toShapeId().getNamespace() == "smithy.synthetic")
+        // enumValue can be derived from enum schemas anyway, so we're removing it from hints
+        .filterNot(_.toShapeId().toString() == "smithy.api#enumValue")
```

Comment:
thy shall not use `.toString` when there is an alternative (in this case, `ShapeId.fromParts("smithy.api", "enumValue")` or probably even `EnumValueTrait.ID` or something

Reply:
now you're just nitpicking :P

Reply:
Nah, just .toString trauma 😄

Reply:
:heavy-nodding: